packages/contracts-bedrock/test/L1/ProtocolVersions.t.sol

```diff
@@ -24,6 +24,15 @@ contract ProtocolVersions_TestInit is CommonTest {
     }
 }
 
+/// @title ProtocolVersions_Version_Test
+/// @notice Test contract for ProtocolVersions `version` function.
+contract ProtocolVersions_Version_Test is ProtocolVersions_TestInit {
+    /// @notice Tests that the version function returns a valid string.
+    function test_version_succeeds() external view {
+        assert(bytes(protocolVersions.version()).length > 0);
```

Comment:
This would be more useful if it would test that the version function returns a valid semver string. As-is, it only tests that we didn't forget to set it to something.

Reply:
This is designed like this in the prompt itself after a conversation we had with Ariel and Kelvin. However, I agree that creating a helper for testing would be probably better